PayPal/src/main/java/com/braintreepayments/api/paypal/vaultedit/PayPalVaultEditAuthRequest.kt

```diff
@@ -1,22 +1,25 @@
 package com.braintreepayments.api.paypal.vaultedit
 
+import com.braintreepayments.api.BrowserSwitchOptions
 import com.braintreepayments.api.core.ExperimentalBetaApi
 
-/**
- * A request used to launch the continuation of the PayPal Edit Vault flow.
- */
 @ExperimentalBetaApi
 sealed class PayPalVaultEditAuthRequest {
 
     /**
      * The request was successfully created and is ready to be launched by [PayPalLauncher]
+     *
+     * @property riskCorrelationId This ID is used to link subsequent retry attempts if payment is declined
      */
     class ReadyToLaunch internal constructor(
-        internal val requestParams: PayPalVaultEditAuthRequestParams
+        val riskCorrelationId: String,
```

Comment:
I think this can be internal - they shouldn't need access to this in the the middle of the flow